icons/mouse-pointer-2.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m4 4 7.07 17 2.51-7.39L21 11.07z" />
+  <path d="M4.037 4.688a.495.495 0 0 1 .651-.651l16 6.5a.5.5 0 0 1-.063.947l-6.124 1.58a2 2 0 0 0-1.438 1.435l-1.579 6.126a.5.5 0 0 1-.947.063z" />
```

Comment:
@karsa-mistmere Is there a reason why every path has 3 decimals? I checked between 2 and 3 decimals, there is no visible difference.

```suggestion
   <path d="M4.04 4.69a.5.5 0 0 1 .65-.65l16 6.5a.5.5 0 0 1-.07.94l-6.12 1.58a2 2 0 0 0-1.44 1.44l-1.58 6.12a.5.5 0 0 1-.94.07z"/>
```

Reply:
@ericfennis just the usual:

<img width="518" alt="image" src="https://github.com/user-attachments/assets/deea31b7-1f2d-4394-b374-167c23e4b4e0">
(3 points vs 2 points)

Reply:
ah check, makes sense!